editor/editor_settings.cpp

```diff
@@ -557,6 +557,8 @@ void EditorSettings::_load_defaults(Ref<ConfigFile> p_extra_config) {
 	hints["editors/3d/navigation_feel/manipulation_translation_inertia"] = PropertyInfo(Variant::FLOAT, "editors/3d/navigation_feel/manipulation_translation_inertia", PROPERTY_HINT_RANGE, "0.0, 1, 0.01");
 
 	// 3D: Freelook
+	_initial_set("editors/3d/navigation_feel/freelook_sensitivity", 0.4);
```

Comment:
This should probably be `editors/3d/freelook/freelook_sensitivity` so you can find it with the other freelook settings.